.github/workflows/jest-testing.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Jest testing suite
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - development
+    paths-ignore:
+      - "**/total-rewards.txt"
```

Comment:
Shouldn't it be deleted?

Reply:
Not sure what this is about, copied from another repo that had the testing set up for Jest. 
Comes from this commit
https://github.com/ubiquity/devpool-directory/commit/ccfcf68e2fdfbdf491bc5d478b63c8f0c5c0eaba
If you know more about it please let me know.